pyproject.toml

```
@@ -102,29 +110,27 @@ exclude = "(?x)(^tests/.*/fixtures | ^src/poetry/core/_vendor)"
 module = [
   # src modules
   'poetry.core.masonry.builders.*',
-  'poetry.core.packages.constraints',
-  'poetry.core.packages.constraints.constraint',
-  'poetry.core.packages.constraints.empty_constraint',
-  'poetry.core.packages.constraints.multi_constraint',
-  'poetry.core.packages.constraints.union_constraint',
   'poetry.core.packages.utils.utils',
   'poetry.core.packages.dependency',
-  'poetry.core.packages.directory_dependency',
-  'poetry.core.packages.file_dependency',
   'poetry.core.packages.package',
   'poetry.core.packages.project_package',
-  'poetry.core.packages.url_dependency',
-  'poetry.core.packages.vcs_dependency',
-  'poetry.core.pyproject.*',
   'poetry.core.semver.*',
+  'poetry.core.toml.*',
   'poetry.core.version.*',
   # test modules
+  'tests.conftest',
+  'tests.integration/test_pep517',
```

should this be 
```suggestion
  'tests.integration.test_pep517',
```
?

---

Either seems to work fine, though I agree that a dot would be consistent with all the others.

This line is moved, rather than introduced by this MR.